### PR TITLE
KFSPTS-16564 clean dashes out of ACH bank accounts before processing

### PR DESCRIPTION
**Pull request by jhulslander**
This is an update to the previous PR https://github.com/CU-CommunityApps/cu-kfs/pull/855 .  We needed to add support for cleaning spaces

### CODE REVIEW

File: src/test/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImplTest.java [hunk 1]
```diff
@@ -724,5 +724,69 @@ public void markRowAsInvalid() {
...
+        validateCleanPayeeACHAccountExtractDetail(" 1 23  4 5 6 ", "123456");
+    }
+    
+    @Test
+    public void testCleanPayeeACHAccountExtractDetailWithSpacesAndDashes() {
+        validateCleanPayeeACHAccountExtractDetail(" 1 2----3  4 - 5 6-7 ", "1234567");
+    }
+    
+    @Test
+    public void testCleanPayeeACHAccountExtractDetailWithSpacesAndDashesAndNumbers() {
```
Comment (cah292) [inline src/test/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImplTest.java]: Should this unit test method be renamed from "...AndNumbers" to "...AndLetters", to more accurately reflect the non-numeric characters that are being tested by this test case?

File: src/test/resources/edu/cornell/kfs/pdp/batch/service/fixture/pdp_ach_test_mixGoodBadLines.csv [hunk 2]
```diff
@@ -1,7 +1,7 @@
 EMPL_ID,NET_ID,LAST_NAME,FIRST_NAME,PAYMENT_TYPE,BALANCE_ACCT_,COMPLETE_DT,BANK_NAME,ROUTING_NO,ACCOUNT_NO,ACCOUNT_TYPE
 1122111,laa111,Doe1,John,Directly Deposited,0,01/01/2016,First Bank,109876543,44333222111,Loan
 4455666,jd8888,Doe,Jane,Direct Deposit,1,01/01/2016,Second Bank,666777888,66555444333,Savings
-4455666,jd8888,Doe,Jane,Direct Deposit,1,01/01/2016,First Bank,666777888,6655-444-33,Savings
+4455666,jd8888,Doe,Jane,Direct Deposit,1,01/01/2016,Second Bank,666777888,665-55-444-3--33,Savings
```
Comment (cah292) [inline src/test/resources/edu/cornell/kfs/pdp/batch/service/fixture/pdp_ach_test_mixGoodBadLines.csv]: I see that some of these test values were updated to include dashes, as part of the new testing. Should any of these include spaces as well?

File: src/test/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImplTest.java (hunk 1 above)
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImplTest.java]: oh, yes, I will change

**Comment by jhulslander** [comment]
I've made some changes based on the code review, this is ready for review again

**Pull request merged**